### DISCUSSION TITLE
Merging multiple personal accounts

### DISCUSSION THREAD

**Discussion by ghost** [community/community#36348] [New to GitHub]
I want to move my work Git Commit History to my personal account. 

However, after checking some github documentation about merging accounts, this should be done as the following,
Going to account -> settings -> emails -> add email.

When I add my email, it says `Error adding xyz@gmail.com: email is already in use` 

So that I was not able to add my personal account to my work account merge them, have I done any mistake?!!

**Comment by airtower-luna** [comment]
An email address can only be attached to one account at a a time. You'll have to remove the address from the work account (or delete the account) *before* you can add the address to your personal account.

**Comment by ghost** [comment]
Hi @airtower-luna , thanks for your time. 

Based on the documentation below, this should be possible to merge 2 accounts. However, if there is only one email for an account at once, this means we are moving the accounts and not Merging accounts. Is this the goal, to move account owner to many accounts(emails)?
<img width="739" alt="Screenshot 2022-10-20 at 16 04 30" src="https://user-images.githubusercontent.com/25284910/196986548-093d972f-5018-4669-b7ac-ec22b52da242.png">

**Comment by najinaser** [comment]
What I understood at the end, there is nothing such as Merge 2 profiles. But you can move repositories. Also, you can change the primary email to a new email address. The new email address should not have a github account.

**Comment by ossanna16** [comment]
Hey there @ghost and thank you for asking a good question! I'm glad you found our community 🙂
